src/ORM/Rule/IsUnique.php

```diff
@@ -53,6 +53,11 @@ public function __invoke(EntityInterface $entity, array $options)
             return true;
         }
 
+        $allowMultipleNulls = true;
+        if (isset($options['allowMultipleNulls'])) {
```

Comment:
@dereuromark I guess it's bad practise to presume that the user won't accidentally enable this, so +1 from me. If others agree, I don't mind _trying_ to change it to a constant.